### PR TITLE
[ResourceBundle] Added return doc statement

### PR DESCRIPTION
**Pull request by dragosprotung**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | no
| License         | MIT
| Doc PR          | yes

The only reason I added this is because the implementation for this interface does return the event

### CODE REVIEW

File: src/Sylius/Bundle/ResourceBundle/Controller/EventDispatcherInterface.php [hunk 1]
```diff
@@ -22,6 +23,7 @@
      * @param string $eventName
      * @param RequestConfiguration $requestConfiguration
      * @param ResourceInterface $resource
+     * @return ResourceControllerEvent
```
Comment (Zales0123) [inline src/Sylius/Bundle/ResourceBundle/Controller/EventDispatcherInterface.php]: Blank line should be added before every ``@return``.

**Comment by Zales0123** [comment]
Hey @dragosprotung, this is a good reason :smile: One thing - "Doc PR" row in PR description's table is to place corresponding PR from [SyliusDocs](https://github.com/Sylius/Sylius-Docs) ;) However, :+1: for creativity ;)

File: src/Sylius/Bundle/ResourceBundle/Controller/EventDispatcherInterface.php (hunk 1 above)
Comment (dragosprotung) [inline src/Sylius/Bundle/ResourceBundle/Controller/EventDispatcherInterface.php]: Updated

**Comment by dragosprotung** [comment]
@Zales0123 Will remember it for next PR :)

**Comment by dragosprotung** [comment]
I pushed more stuff on my master branch and removed it afterwards but travis ran against an inexisting commit. I will create new PR.

Sorry for the confusion

**Pull request closed without merging**